core/services/ocr/contract_tracker_test.go

```diff
@@ -108,14 +107,6 @@ func newContractTrackerUni(t *testing.T, opts ...interface{}) (uni contractTrack
 func Test_OCRContractTracker_LatestBlockHeight(t *testing.T) {
 	t.Parallel()
 
-	t.Run("on L2 chains, always returns 0", func(t *testing.T) {
-		uni := newContractTrackerUni(t, v2.ChainOptimismMainnet(t))
```

Comment:
Ideally, we don't want to remove that. If it's as simple as adding a ChainMetisMainnet instead and using it do it. If it requires changing more files then you can remove it for now.

Reply:
It needs a bit more addition not s single line addition, also I removed it because this assumption is not right anymore, semantically